contracts/standard/arbitration/MultipleArbitrableTransaction.sol

```diff
@@ -0,0 +1,320 @@
+        
+        transaction.lastInteraction = now;
+        if (transaction.sellerFee < arbitrationCost) { // The partyA still has to pay. This can also happens if he has paid, but arbitrationCost has increased.
+            transaction.status = Status.WaitingSeller;
+            HasToPayFee(_transactionId, Party.Seller);
+        } else { // The partyA has also paid the fee. We create the dispute
+            raiseDispute(_transactionId, arbitrationCost);
+        }
+    }
+    
+    /** @dev Create a dispute. UNTRUSTED.
+     *  @param _transactionId The index of the transaction.
+     *  @param _arbitrationCost Amount to pay the arbitrator.
+     */
+    function raiseDispute(uint _transactionId, uint _arbitrationCost) internal {
+        Transaction storage transaction = transactions[_transactionId];
+        transaction.status = Status.DisputeCreated;
+        transaction.disputeId = transaction.arbitrator.createDispute.value(_arbitrationCost)(AMOUNT_OF_CHOICES,transaction.arbitratorExtraData);
+        transaction.disputed = true;
+        disputeTxMap[keccak256(transaction.arbitrator, transaction.disputeId)] = _transactionId;
+        Dispute(_transactionId, transaction.arbitrator, transaction.disputeId,RULING_OPTIONS);
+    }
+    
+    /** @dev Reimburse partyA if partyB fails to pay the fee.
+     *  @param _transactionId The index of the transaction.
+     */
+    function timeOutBySeller(uint _transactionId) {
+        Transaction storage transaction = transactions[_transactionId];
+        require(msg.sender == transaction.seller);
+
+
+        require(transaction.status==Status.WaitingBuyer);
+        require(now>=transaction.lastInteraction+transaction.timeout);
+        
+        executeRuling(transaction.disputeId, SELLER_WINS);
+    }
+    
+    /** @dev Pay partyB if partyA fails to pay the fee.
+     *  @param _transactionId The index of the transaction.
+     */
+    function timeOutByBuyer(uint _transactionId) {
+        Transaction storage transaction = transactions[_transactionId];
+        require(msg.sender == transaction.buyer);
+
+
+        require(transaction.status == Status.WaitingSeller);
+        require(now>=transaction.lastInteraction+transaction.timeout);
+        
+        executeRuling(transaction.disputeId,BUYER_WINS);
+    }
+    
+    /** @dev Submit a reference to evidence. EVENT.
+     *  @param _transactionId The index of the transaction.
+     *  @param _evidence A link to an evidence using its URI.
+     */
+    function submitEvidence(uint _transactionId, string _evidence) {
+        Transaction storage transaction = transactions[_transactionId];
+        require(msg.sender == transaction.buyer || msg.sender == transaction.seller);
+        
+        require(transaction.status>=Status.DisputeCreated);
+        Evidence(_transactionId, transaction.arbitrator,transaction.disputeId,msg.sender,_evidence);
```

Comment:
We can remove the `_transactionId` which will make it consistent with other `Arbitrable` contracts.